src/fides/api/api/v1/endpoints/consent_request_endpoints.py

```diff
@@ -101,12 +101,12 @@ def _filter_consent(
     )
 
     if identity:
-        hashed_identity = ProvidedIdentity.hash_value(value=identity)
+        identity_hashes = ProvidedIdentity.hash_value_for_search(identity)
         identities: Set[str] = {
             identity[0]
             for identity in ProvidedIdentity.filter(
                 db=db,
-                conditions=(ProvidedIdentity.hashed_value == hashed_identity),
+                conditions=(ProvidedIdentity.hashed_value.in_(identity_hashes)),
```

Comment:
This is the main change to be able support the "on the fly" cutover to the new SHA-256 hashes. Instead of checking that the `hashed_value` is equal to a `hashed_identity` we check that it matches either the bcrypt hash or the SHA-256 hash. Each record can be found by one of those two hashes as the migration is in progress but `hash_value_for_search` will only compute the SHA-256 hash after the migration is complete and no longer compute the expensive bcrypt hash.